### PR TITLE
Feature 001

### PR DESCRIPTION
**Pull request by alchan1205**
cr plz

### CODE REVIEW

**Comment by alchan1205** [comment]
@alchan1205 test it

**Comment by robin1205** [comment]
@alchan1205 check it

File: src/main/java/sk/testing/skeleton/Main.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
-package sk.testing.skeleton;
-
-public class Main {
-	
-}
+package sk.testing.skeleton;
+
+public class Main {
+//	blah blah blah
```
Comment (robin1205) [inline src/main/java/sk/testing/skeleton/Main.java]: check again

Comment (alchan1205) [reply src/main/java/sk/testing/skeleton/Main.java]: why

Comment (robin1205) [inline src/main/java/sk/testing/skeleton/Main.java]: dont use single comment

**Comment by alchan1205** [comment]
from jenkins

**Pull request merged**